website/app.js

```diff
@@ -3,35 +3,65 @@ const apiEndPoint = "api.openweathermap.org";
 const apiKey = "081a059f88749d9d3c5a61e73985ef20";
 const country = "us";
 
-// Create a new date instance dynamically with JS
-let d = new Date();
-let newDate = d.getMonth() + "." + d.getDate() + "." + d.getFullYear();
-
-document.getElementById("generate").addEventListener("click", async () => {
-  const zipValue = document.getElementById("zip").value;
-  const feelings = document.getElementById("feelings").value;
-  await getData(zipValue, country, feelings);
-  showData();
-});
+/**
+ * Gets today's date and parses it as US String: MM/DD/YYYY
+ * @return {string} - The date to be showned.
+ */
+const getTodaysDataAsString = () => {
```

Comment:
Encapsulating the get date behavior on a function will help have the code better organized and when we add webpack we will be able to move these kind of functions to an utilities file.